README.md

```diff
@@ -271,13 +271,44 @@ To try out Carbon, you can use the Carbon explorer to interpret Carbon code and
 print its output. You can try it out immediately at
 [compiler-explorer.com](http://carbon.compiler-explorer.com/).
 
-Because Carbon is an early, experimental project we don't yet have releases you
-can download and try out locally, you'll instead need to build any tools
-yourself from source. We expect to have packaged releases you can try out when
-we reach our
+Carbon is an early, experimental project, and so we only have very experimental
+nightly releases of the Carbon toolchain available to download, and only on
```

Comment:
FWIW, this feels a bit odd coming after the "explorer" advice above.

Reply:
I'd like to keep opening with the compiler-explorer link that folks can just dive into.

But I agree we need a transition in the text... I've tried to reword both sections a bit to emphasize that the above is just about trying stuff out directly in the browser. I've also removed the explorer reference from a local build below to focus in on the toolchain where we would potentially be interested in contributions.

PR description also updated a bit to reflect the changes here.

Reply:
NB, I think you might've forgotten to push the commit for this.

Reply:
I... think it is pushed? Maybe it wasn't as much of an adjustment as you were looking for?